target/xilinx/scripts/impl_ip.tcl

```diff
@@ -162,6 +210,33 @@ switch $proj {
                     CONFIG.C0.DDR4_AxiSelection {true} \
                     ] [get_ips $proj]
             }
+            vcu118 {
+                set_property -dict [list \
+                    CONFIG.C0.DDR4_Clamshell {false} \
+                    CONFIG.C0_DDR4_BOARD_INTERFACE {Custom} \
```

Comment:
Why not choosing one of the ddr4 interfaces proposed by Vivado? This makes the [constraint file](https://github.com/pulp-platform/cheshire/blob/main/target/xilinx/constraints/vcu128.xdc) less verbose.